src/Access/RowPolicy.h

```diff
@@ -14,6 +14,8 @@ namespace DB
   */
 struct RowPolicy : public IAccessEntity
 {
+    static constexpr char DATABASE_MARK[] = "*";
```

Comment:
I suggest naming it "ANY_TABLE_MARK" or something similar, because it is used exclusively to check against table names.